app/views/candidate_interface/personal_details/immigration_status/edit.html.erb

```diff
@@ -1,4 +1,5 @@
-<% content_for :title, title_with_error_prefix(t('page_titles.immigration_status'), @form.errors.any?) %>
+<% content_for :title, title_with_error_prefix(t("page_titles.#{@form.eu_nationality? ? 'immigration_status' : 'visa_or_immigration_status'}"), @form.errors.any?) %>
```

Comment:
This pattern appears again. Can we improve on this?

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/9012/commits/b0dd52986d98f64b073f8d0495c71f2ca1b86d5a